src/dashboards.js

```diff
@@ -10,18 +10,17 @@ const http = require('http');
 const superagent = require('superagent');
 
 
-const CONFIG_PATH = path.join(__dirname,"config.json");
+const CONFIG_PATH = path.join(__dirname, "config.json");
 
 
-function getConfig() {
+function getConfig(CONFIG_PATH=CONFIG_PATH) {
```

Comment:
what JS closure concept, it is trick to tell CONFIG_PATH is local variable, method variable of global variable to non-js developer. 

let's use clear variable name for each scope to avoid that.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures#closure